src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.html

```diff
@@ -18,15 +18,16 @@
 
     <button type="button"
             class="navbar-toggler"
-            (click)="isCollapsed = !isCollapsed">
+            (click)="toggleRightSidebar()">
       <span i18n
             class="sr-only">Toggle navigation</span>
       <span class="">
         <i class="fa fa-navicon fa-lg"></i>
       </span>
     </button>
 
-    <div class="collapse navbar-collapse">
+    <div class="collapse navbar-collapse show"
```

Comment:
It seems `show` is not nede here.

Reply:
Yeah! I just checked this, and I don't think it's needed :sweat_smile: . Thank you for pointing that out